apps/prairielearn/python/prairielearn/misc_utils.py

```diff
@@ -33,7 +36,8 @@ def index2key(i: int) -> str:
     """
     Use when generating ordered lists of the form ['a', 'b', ..., 'z', 'aa', 'ab', ..., 'zz', 'aaa', 'aab', ...]
 
-    Returns alphabetic key in the form [a-z]* from a given integer (i = 0, 1, 2, ...).
+    Returns:
+        alphabetic key in the form `[a-z]*` from a given integer (i = 0, 1, 2, ...).
```

Comment:
```suggestion
        Alphabetic key in the form `[a-z]*` from a given integer (i = 0, 1, 2, ...).
```

Reply:
Resolved in [163dfab](https://github.com/PrairieLearn/PrairieLearn/pull/11758/commits/163dfab708fb88fca0820d6e6f061f20e2cddebe).